tinygrad/runtime/ops_clang.py

```diff
@@ -14,12 +14,10 @@
 class ClangProgram:
   def __init__(self, name:str, prg:str):
     prg = CLANG_PROGRAM_HEADER + prg
-    # TODO: is there a way to not write this to disk?
-    fn = f"{tempfile.gettempdir()}/clang_{hashlib.md5(prg.encode('utf-8')).hexdigest()}.{args['ext']}"
+    # /run/shm is a in-mem tmpfs, so this is fast
+    fn = f"/run/shm/clang_{hashlib.md5(prg.encode('utf-8')).hexdigest()}.{args['ext']}"
```

Comment:
is `/run/shm` standard? it doesn't exist on my system, but `/dev/shm` does

Reply:
Standards changed but I'm not sure which version it happened.

Reply:
https://askubuntu.com/questions/169495/what-are-run-lock-and-run-shm-used-for